global-script.lua

```diff
@@ -304,6 +324,13 @@ function onLoad(saved_data)
     if saved_data ~= "" then
         local loaded_data = JSON.decode(saved_data)
         gameStarted = loaded_data.gameStarted
+        if loaded_data.playerBlocks then
+            playerBlocks = loaded_data.playerBlocks
+            playerTables = loaded_data.playerTables
```

Comment:
from the looks of it `playerTables` doesn't appear to be used either after a game has started

Reply:
See the earlier comments about the potential of a save existing from before the game started